releases.yml

```diff
@@ -31,8 +31,13 @@ releases:
         images: []
         rpms:
         - distgit_key: microshift
-          why: Pin microshift to assembly
+          why: microshift team askes to use a specified commit
           metadata:
+            content:
+              source:
+                git:
+                  branch:
+                    target: f446eda4f26ad8430f6452891010e7a4f8ee5e43
             is:
               el9: microshift-4.14.0~rc.5-202310121510.p0.g6a73516.assembly.rc.5.el9
```

Comment:
Not sure if this needs to be removed?

Reply:
Yeah I'd say use the pr branch to run the job, and then update this section as well with the new builds